src/components/users/UserCards.js

```diff
@@ -0,0 +1,96 @@
+        },
+        card: {
+            padding: 18,
+            borderRadius: 18
+        },
+        cardContent:{
+            marginBottom: 8
+        },
+        linkText: {
+            textDecoration: `none`,
+            color: `white`,
+            textAlign: 'center'
+
+        }
+    }
+})
+
+export default function UserCards({handleDelete, handleEdit}) {
+    const classes = useStyles();
+    const {filteredUsers} = useContext(UserContext)
+
+
+    return (
+        <div>
+            <Grid align="center" item xs={12} sm={12} md={12}>
+                <a className={classes.linkText} href="/create/user">
+                    <Button variant="contained" size="large" color="secondary">Create User</Button>    
+                </a>
+            </Grid>
+            <Grid className={classes.cardContainer} container spacing={3}>
+                {
+                    filteredUsers.map(user => {
+                        if(user.isActive){
+                            return(  
+                                <Grid key={user._id} item xs={12} sm={6} md={4}>
+                                    <Card elevation={10} className={classes.card}>
+                                        <CardHeader 
+                                            avatar={<Avatar className={classes.userAvatar} >{user.name ? user.name[0].toUpperCase() : ' '}</Avatar>}
+                                            title={<Typography variant="h6">{user.name}</Typography>}
+                                            subheader={user.email}
+                                        />
+                
+                                        <CardContent>
+                                            <Typography className={classes.cardContent}><strong>City:</strong> {user.city}</Typography>
+                                            <Typography className={classes.cardContent}><strong>Address:</strong> {user.address}</Typography>
+                                            <Typography><strong>Phone:</strong> {user.phone}</Typography>
+                                            <Typography><strong>Is Active:</strong> {user.isActive ? 'Yes' : 'No'}</Typography>
+                                        </CardContent>
+    
+                                        <CardActions>
+                                            <Button 
+                                                id={user._id} 
+                                                variant="contained" size="small" color="primary" 
+                                                onClick={(e) => {handleEdit(e)}}
+                                            >
+                                            EDIT
+                                            </Button>
+                                            <Button 
+                                                id={user._id} 
+                                                variant="contained" 
+                                                size="small" 

```

Comment:
Can we further make this return statement into one component and use that here ?

Reply:
@waleedbinalim please resolve this as well so I can merge it.

Reply:
Trying to figure how to set ID's on it when the button groups are in a separate component

Reply:
should I do the same for order cards as well?